patches/net/minecraft/world/item/CreativeModeTab.java.patch

```diff
@@ -230,6 +230,20 @@
 +            java.util.stream.Stream.of(tabs).map(net.minecraft.resources.ResourceKey::location).forEach(this.tabsAfter::add);
 +            return this;
 +        }
++
++        /**
++         * Helper to set this tabs contents to everything in the supplied {@link net.neoforged.neoforge.registries.DeferredRegister}.
```

Comment:
Wording: This does not set tab contents, but rather appends/adds more entries from the DR.